docs/use-cases/Intel-Discrete-GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,277 @@
+# Using Intel Discrete GPU device with Kata Containers
+
+This guide covers the use case for passing Intel Discrete GPUs to Kata.
+These include the Intel® Data Center GPU Max Series and Intel® Data Center GPU Flex Series.
+For integrated GPUs please refer to [Integrate-Intel-GPUs-with-Kata](../Intel-GPU-passthrough-and-Kata.md) 
+
+> **Note:** These instructions are for a system that has an x86_64 CPU.
+
+An Intel Discrete GPU can be passed to a Kata Container using GPU passthrough. 
+as well as SRIOV passthrough.
+
+In Intel GPU pass-through mode, an entire physical GPU is directly assigned to one VM. 
+In this mode of operation, the GPU is accessed exclusively by the Intel driver running in
+the VM to which it is assigned. The GPU is not shared among VMs.
+
+With SRIOV mode, it is possible to pass a Virtual GPU instance to a virtual machine.
+With this, multiple Virtual GPU instances can be carved out of a single physical GPU 
+and be passed to different VMs, allowing the GPU to be shared.
+
+| Technology | Description |
```

Comment:
It might be worth adding some extra columns showing the proportion of the GPU that is available in the different contexts. Something like:

| Technology | Proportion of GPU shared to VM | Proportion of GPU accessible to host | Description |
|-|-|-|-|
| GPU passthrough | 100% | 0% | Physical GPU assigned to a single VM |
| SR-IOV passthrough | varies | varies | Physical GPU shared by multiple VMs |

If so, it does raise the question, _"What is the minimum GPU % I can pass to a single VM when using SR-IOV?"_ so it might be worth stating that as a [note](https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#notes).

Reply:
I think adding the extra column is confusing in this case. Since we are dealing with passthrough, we should be conncerned about the proportion shared with the VM.